### DISCUSSION TITLE
Possibility to create conditional Categories?

### DISCUSSION THREAD

**Discussion by dlisjak** [strapi/strapi#8716] [Legacy Topics]
Hello. I am facing a problem I cannot seem to solve...
I am creating Collection Types (Estate) that need to have a Type (Appartment, House, Land...). After that I would need to also add another Category based on the Type that was selected. Meaning if Estate has a type of Appartment, I would like a select box with the options: (New, Used). If Land is selected, I would like to have the options: (Not built, built) etc...
Does this make sense?
Thank you for the help.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
